### ISSUE TITLE
Vulnerability prioritization

### ISSUE THREAD

**Issue by andresghelarducci** [DependencyTrack/dependency-track#2671]
### Current Behavior

Currently DependencyTrack provide CVSS and EPSS  and an analys window where some information can be inserted to analyze the vulnerability

### Proposed Behavior

In order to help vulnerability prioritization can something like SSVC rating  (https://www.cisa.gov/stakeholder-specific-vulnerability-categorization-ssvc) be added to dependencyTrack?
In particular it may be useful to have a way to generate the decision tree  inside dependencytrack

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this enhancement was already requested

**Comment by nscuro** [comment]
@Whatsonyourmind No advertisement of commercial AI slop here please.

**Comment by fahedouch** [comment]
I am working on a [PR](https://github.com/DependencyTrack/hyades-apiserver/pull/1928) that facilitates vulnerability prioritization based on context-aware OWASP scores.

**Comment by Whatsonyourmind** [comment]
@nscuro You are right, and I apologize. My earlier comment on this thread was promotional, and that is exactly the kind of noise that shouldn't be dropped into a maintainer's backlog. I deleted it, and I am stepping out of DependencyTrack issues going forward. Thanks for the direct correction and sorry again for wasting your time.
